src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -29,14 +29,29 @@ const uint32_t major = 1;
 const uint32_t minor = 0;
 const uint32_t patch = 0;
 
+inline std::string
+graph_status_to_string(int status) {
+  switch(status)
+  {
+    case 0: return std::string("stop");

```

Comment:
Please have the status value be an enumeration.

Reply:
done